nearcore/src/state_sync.rs

```diff
@@ -276,28 +296,20 @@ fn get_current_state(
     };
 
     if Some(&new_epoch_id) == was_last_epoch_done.as_ref() {
-        tracing::debug!(target: "state_sync_dump", ?shard_id, ?was_last_epoch_done, ?new_epoch_id, new_epoch_height, ?new_sync_hash, "latest epoch is done. No new epoch to dump. Idle");
```

Comment:
Why remove the comment?

Reply:
I guess it's a sort of unrelated change yeah... But in my opinion this file is just way too verbose in general, and this log line in particular feels kind of unnecessary, since we're printing it on a loop constantly after the state dump is done. If you take a look at logs generated by the state dumper by grepping for "state_sync_dump", it's just kind of unreadable. This deletion is one small step to chip away at that I guess.